Cargo.lock

```diff
@@ -2039,11 +2102,11 @@ dependencies = [
 
 [[package]]
 name = "mbedtls-sys-auto"
-version = "2.28.0"
+version = "2.28.10"
```

Comment:
FYI (Discussed offline): This change is needed as mbedtls-sys-auto 2.28.0 resulted in a build failure when compiled with clang-18 due to an unused variable in C code, and the `-Werror` option being enabled.